app/Community/Concerns/ActsAsCommunityMember.php

```diff
@@ -50,33 +50,33 @@ public function gameListEntries(?string $type = null): HasMany
     /**
      * @return BelongsToMany<User>
      */
-    public function relationships(): BelongsToMany
+    public function relatedUsers(): BelongsToMany
     {
         return $this->belongsToMany(User::class, (new UserRelation())->getTable(), 'user_id', 'related_user_id');
     }
 
     /**
      * @return BelongsToMany<User>
      */
-    public function inverseRelationships(): BelongsToMany
+    public function inverseRelatedUsers(): BelongsToMany
     {
         return $this->belongsToMany(User::class, (new UserRelation())->getTable(), 'related_user_id', 'user_id');
     }
 
     /**
      * @return BelongsToMany<User>
      */
-    public function following(): BelongsToMany
```

Comment:
There's still a reference to `following()` in `compare-progress.blade.php`

Reply:
Fixed in latest.